src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/PopulateLicenceDeletedAtMigration.kt

```diff
@@ -0,0 +1,98 @@
+package uk.gov.justice.digital.hmpps.hmppshdcapi.licences
```

Comment:
Are you planning on integration testing this once its hooked up?

Reply:
yep 😅